### PR TITLE
(PE-35511) Update to tk-webserver-jetty-10

### PR DESCRIPTION
**Pull request by rileynewton**

### CODE REVIEW

File: resources/puppetlabs/puppetdb/benchmark/samples/facts/pg2.vm.json [hunk 1]
```diff
@@ -4798,7 +4798,7 @@
             "apt"
         ],
         [
-            "libjetty9-java",
+            "libjetty10-java",
```
Comment (jonathannewman) [inline resources/puppetlabs/puppetdb/benchmark/samples/facts/pg2.vm.json:4801]: Not sure you wanted to make this change here.

Comment (rileynewton) [reply resources/puppetlabs/puppetdb/benchmark/samples/facts/pg2.vm.json]: Was thinking it's a doc so doesn't matter though not sure about that, reverted it

File: src/puppetlabs/puppetdb/http/query.clj [hunk 2]
```diff
@@ -12,7 +12,7 @@
             [murphy :refer [try!]]
             [puppetlabs.puppetdb.query-eng :as qeng]
             [puppetlabs.puppetdb.query.monitor :as qmon]
-            [puppetlabs.trapperkeeper.services.webserver.jetty9 :as jetty9]
+            [puppetlabs.trapperkeeper.services.webserver.jetty10 :as jetty10]
```
Comment (jonathannewman) [inline src/puppetlabs/puppetdb/http/query.clj:15]: Both requires can be removed.  It was just used for the name spacing, which doesn't exist in jetty10.

Comment (rileynewton) [reply src/puppetlabs/puppetdb/http/query.clj]: Removed this one, which is the other?

**Comment by jonathannewman** [comment]
I think your parent version needs to be updated too.

File: project.clj [hunk 3]
```diff
@@ -177,8 +177,8 @@
                  [puppetlabs/stockpile "0.0.4"]
                  [puppetlabs/structured-logging]
                  [puppetlabs/trapperkeeper]
-                 [puppetlabs/trapperkeeper-webserver-jetty9]
-                 [puppetlabs/trapperkeeper-metrics]
+                 [com.puppetlabs/trapperkeeper-webserver-jetty10]
```
Comment (jonathannewman) [inline project.clj:180]: Don't forget to update the version here too if you want to test!

**Comment by rileynewton** [comment]
> I think your parent version needs to be updated too.

Didn't release a new one yet--seeing if this fixes the issue whenever tk-metrics manages to release.

**Review by jonathannewman — APPROVED** [review]

**Pull request merged**